gapis/gfxapi/templates/vulkan_gfx_api_extras.tmpl

```diff
@@ -377,6 +377,32 @@ bool Vulkan::replayGetFenceStatus(Stack* stack, bool pushReturn) {
     }
 }
 ¶
+bool Vulkan::replayGetEventStatus(Stack* stack, bool pushReturn) {
+    auto expected = gapir::Vulkan::VkResult(stack->pop<uint32_t>());
+    auto event = stack->pop<uint64_t>();
+    auto device = static_cast<size_val>(stack->pop<size_val>());
+    if (stack->isValid()) {
+        GAPID_INFO("vkGetEventStatus(%" PRIsize ", %" PRIu64 ")", device, event);
+        if (mVkDeviceFunctionStubs.find(device) != mVkDeviceFunctionStubs.end() &&
+        mVkDeviceFunctionStubs[device].vkGetEventStatus) {
+            VkResult return_value;
+            do {
+                return_value = mVkDeviceFunctionStubs[device].vkGetEventStatus(device, event);
+            } while (return_value != expected);
```

Comment:
You don't want this.

If in the trace, the event has not yet been set, but in the replay it has already been set, we will loop forever.